discovery-infra/tests/test_logs.py

```diff
@@ -10,114 +10,80 @@
 
 
 class TestDownloadLogs(BaseTest):
-    def setup_hosts(self, cluster_id, api_client, nodes):
-        '''setup nodes from ISO image and wait until they are registered'''
-        # Generate and download cluster ISO
-        self.generate_and_download_image(cluster_id=cluster_id, api_client=api_client)
-        # Boot nodes into ISO
-        nodes.start_all()
-        # Wait until hosts are discovered and update host roles
-        self.wait_until_hosts_are_discovered(cluster_id=cluster_id, api_client=api_client)
+    def _install_cluster_and_wait(self, cluster):
```

Comment:
why not to have this func in cluster?
add one that does this with flags as:
def _install_cluster_and_wait(self, wait_for_hosts=True, wait_for_cluster_inst=True)